Realm/RLMRealm.h

```diff
@@ -518,6 +518,31 @@ typedef void (^RLMMigrationBlock)(RLMMigration *migration, NSUInteger oldSchemaV
 + (void)setSchemaVersion:(NSUInteger)version withMigrationBlock:(RLMMigrationBlock)block;
 
 /**
+ Get the schema version for a Realm at a given path.
+
+ @param realmPath   Path to a Realm file
+ @param error       If an error occurs, upon return contains an `NSError` object
+                    that describes the problem. If you are not interested in
+                    possible errors, pass in `NULL`.
+
+ @return            The version of the Realm at `realmPath`
```

Comment:
or RLMNotVersioned if the version cannot be read.

Also RLMNotVersioned needs to be moved to a public header.